static/css/around-map.css

```diff
@@ -295,63 +295,34 @@ footer {
   background-color: #00BF6BEE;
 }
 
-/* ===================================== */
-/* OSM & Preact build CSS into html tag  */
-
-.mapboxgl-canvas-container {
-  position: fixed !important;
-}
-
-.super-card {
-  border: 0.2rem solid #3ba77c !important;
-}
-
-.first-card {
-  order: 0 !important;
-}
-
-.current-marker {
-  border: 0.4rem solid #3ba77c !important;
-  z-index: 1;
-}
-
 .visible {
-  display: flex !important;
+  display: flex!important;
 }
 
-/* ===================================== */
-
-.super-marker {
-  background-image: url('/static/images/marker-green.png');
-  background-size: cover;
-  cursor: pointer;
-  width: 2rem;
-  height: 3rem;
-  border-radius: 0;
-  background-color: transparent;
-  border: transparent;
-  z-index: 2;
-  left: -0.4rem;
-  top: -2rem;
+.first-card {
```

Comment:
jsut curious, but why setting a class instead of using `:first-of-type` or `:first-child`